README.md

```diff
@@ -5,18 +5,19 @@ Freemodel Resource Name (frn) - go package to manage universal ids
 
 ### Tags
 
-| annotation             | example                                                    | description                                      |
-|:-----------------------|:-----------------------------------------------------------|:-------------------------------------------------|
-| `frn=project`          | {namespace}:{service}:project:{id}                         | require unary form                               |
-| `frn=project/`         | {namespace}:{service}:project:{id}:{child-type}:{child-id} | require compound form; unary form not acceptable |
-| `frn=/contract`        | {namespace}:{service}:{type}:{id}:contract:{child-id}      | require child type                               |
-| `frn=project/contract` | {namespace}:{service}:project:{id}:contract:{child-id      | require parent and child                         |
-
+| annotation                 | example                                                    | description                                      |
+|:---------------------------|:-----------------------------------------------------------|:-------------------------------------------------|
+| `frn=project`              | {namespace}:{service}:project:{id}                         | require unary form                               |
+| `frn=project/`             | {namespace}:{service}:project:{id}:{child-type}:{child-id} | require compound form; unary form not acceptable |
+| `frn=/contract`            | {namespace}:{service}:{type}:{id}:contract:{child-id}      | require child type                               |
+| `frn=/contract/key`        | {namespace}:{service}:{type}:{id}:contract:{child-id}/key  | require child type with path head, key           |
+| `frn=project/contract`     | {namespace}:{service}:project:{id}:contract:{child-id}     | require parent and child                         |
+| `frn=project/contract/key` | {namespace}:{service}:project:{id}:contract:{child-id}/key | require parent and child and path head, key      |
```

Comment:
it might be good to have an example of what the `key/value` aspects look like once completed. I just see the key here, and not the value.

But I could be misunderstanding how this is supposed to look? 

Which would reinforce the benefit of having a clear example of what a full key looks like.

Reply:
also good point